client/components/tinymce/plugins/embed/docs/example.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { identity, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import { EmbedDialog } from '../dialog';
+
+export default class EmbedDialogExample extends PureComponent {
+	state = {
+		embedUrl: 'https://www.youtube.com/watch?v=R54QEvTyqO4',
+		showDialog: false,
+	};
+
+	openDialog = () => this.setState( { showDialog: true } );
+
+	onCancel = () => {
+		this.setState( { showDialog: false } );
+	};
+
+	onUpdate = ( newUrl ) => {
+		this.setState( {
+			embedUrl: newUrl,
+			showDialog: false,
+		} );
+	};
+
+	render() {
+		return (
+			<Card>
+				<Button onClick={ this.openDialog }>
+					Open Embed Dialog
+				</Button>
+
+				<EmbedDialog
+					embedUrl={ this.state.embedUrl }
+					isVisible={ this.state.showDialog }
+					onCancel={ this.onCancel }
+					onUpdate={ this.onUpdate }
+					translate={ identity }
```

Comment:
This might be a little misleading to mention in the example. I'm not sure what others think but I'd leave it out since the reader of the example shouldn't need to consider it.

Reply:
Are you referring to `translate`? I added it here since it's a `required` prop.

https://github.com/Automattic/wp-calypso/pull/18064#discussion_r140893869

Reply:
Ahh, I'm with you - I didn't spot that the base component was being imported directly :)
That said, should we be recommending it's use in the example or should we recommend using the enhanced component ( `export default` )?
I could be wrong but I think the only reason one would need to import the base component would be for test purposes.

Reply:
Ah, I get what you're saying now, and you're totally right. The example should use the `default`, localized component, since that's what would normally be used when adding new uses of the component elsewhere in Calypso. Great catch, thanks!